server/network/aoprotocol.py

```diff
@@ -640,9 +640,7 @@ def net_cmd_ct(self, args):
                 arg = spl[1][:256]
             try:
                 called_function = f'ooc_cmd_{cmd}'
-                if cmd == 'help' and arg != '':
-                    self.client.send_ooc(commands.help(f'ooc_cmd_{arg}'))
```

Comment:
i had been running across a complication when i tried to do it directly through the ooc_cmd_help handler - i don't remember what it was, so i did this and called it a day